.github/ISSUE_TEMPLATE/bug_report_template.md

```diff
@@ -12,43 +12,34 @@ assignees: ''
 Thank you for submitting an issue. Please refer to our [issue policy](https://www.github.com/voxel51/fiftyone/blob/develop/ISSUE_POLICY.md) for information on what types of issues we address.
 
 1. Please fill in this template to ensure a timely and thorough response
-2. Place an "x" between the brackets next to an option if it applies. For example:
+2. Remove the section instructions but leave the section header
+3. Place an "x" between the brackets next to an option if it applies. For example:
     -   [x] Selected option
-3. **Please delete everything above this line before submitting the issue**
+4. **Please delete everything above this line before submitting the issue**
 
-### System information
+### Describe the problem
 
--   **OS Platform and Distribution** (e.g., Linux Ubuntu 16.04):
--   **Python version** (`python --version`):
--   **FiftyOne version** (`fiftyone --version`):
--   **FiftyOne installed from** (pip or source):
+Describe the problem clearly here. Include descriptions of the expected behavior and the actual behavior.
 
-### Commands to reproduce
+### Code to reproduce issue
 
-As thoroughly as possible, please provide the Python and/or shell commands used to encounter the issue. Application steps can be described in the next section.
+Provide a reproducible test case that is the bare minimum necessary to generate the problem. Include a short video or screenshot if the bug is in the app.
 
 ```
-# commands here
+# commands and/or screenshots here
 ```
 
-### Describe the problem
-
-Describe the problem clearly here. Include descriptions of the expected behavior and the actual behavior.
-
-### Code to reproduce issue
+### System information
 
-Provide a reproducible test case that is the bare minimum necessary to generate the problem.
+-   **OS Platform and Distribution** (e.g., Linux Ubuntu 16.04):
+-   **Python version** (`python --version`):
+-   **FiftyOne version** (`fiftyone --version`):
+-   **FiftyOne installed from** (pip or source):
 
 ### Other info/logs
 
 Include any logs or source code that would be helpful to diagnose the problem. If including tracebacks, please include the full traceback. Large logs and files should be attached. Please do not use screenshots for sharing text. Code snippets should be used instead when providing tracebacks, logs, etc.
 
-### What areas of FiftyOne does this bug affect?
```

Comment:
I don't think this section provides any valuable information to us. It will likely be obvious to maintainers by the description and reproduction steps. And the user might not know distinction between app and server or core and server, for example.
If we wanted to bucket issues into categories then we should use labels.